test/test_torch.py

```diff
@@ -5667,6 +5667,54 @@ def test_fill_diagonal(self):
         e1.fill_diagonal_(v, wrap=True)
         self.assertEqual(e1, e2)
 
+    def test_batch_norm_cpu_inference(self):
```

Comment:
it would be good to test this on float32s and float64s.  You can search around the test framework for examples of that.

But since you are specifying magic values you should check that the magic works on all the combinations we care about.